src/main.cpp

```diff
@@ -104,6 +122,11 @@ void setup()
 
 void loop()
 {
+    // delay(10);
+    // uint16_t value = generateSineValue(millis(),1500/2,0.5,0,0);
+    // pedals.read_pedal_values_debug(value);
+    // pedals.run_pedals();
+    // pedals.calculate_torque(dummy_motor_speed,dummy_max_torque);
```

Comment:
so this would let you test the calculate torque functions and such without having a running car?

Reply:
yeah this is what I used to test the pedalSensor class and refactored calculate_torque